developers.diem.com/docs/sdks/overview.md

```diff
@@ -12,16 +12,17 @@ Select a language to access its approved Diem resource package on GitHub:
 
 <CardsWrapper cardsPerRow={2}>
   <SDKCard
+    docs="https://javadoc.io/doc/com.diem/client-sdk-java/latest/index.html"
     icon="/img/docs/sdk-java.png"
     sdk="https://github.com/libra/client-sdk-java"
   />
   <SDKCard
-    docs="https://godoc.org"
+    docs="https://godoc.org/github.com/diem/client-sdk-go"
```

Comment:
This is a weird link structure, but it seems to work.

Reply:
"github.com/diem/client-sdk-go" is the package import path, in golang, codebase URL is your package import path. published package is your codebase, how simple it is :)